clients/src/test/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinatorTest.java

```diff
@@ -458,6 +482,10 @@ public boolean matches(AbstractRequest body) {
 
     private void awaitFirstHeartbeat(final AtomicBoolean heartbeatReceived) throws Exception {
         mockTime.sleep(HEARTBEAT_INTERVAL_MS);
+        // Awake the heartbeat thread
```

Comment:
It speeds up the tests because it avoids waiting the retry back off. I can remove it if you prefer?
The new test case does not use it because it calls the method pollHeartbeat.